#ifndef COMPONENT_MSGS_H
#define COMPONENT_MSGS_H

#include <component_lib/KondoServoCommandMsg.h>
#include <component_lib/MotorCommandMsg.h>
#include <component_lib/KondoServoFeedbackMsg.h>
#include <component_lib/MotorFeedbackMsg.h>
#include <component_lib/KondoServoMsg.h>
#include <component_lib/MotorMsg.h>

#endif
